docs/OnnxReleases.md

```diff
@@ -45,13 +63,52 @@ repository=https://test.pypi.org/legacy/
+
+## PyPI package verification
+**Test ONNX itself**
+* Test the PyPI package installation with different combinations of various Python versions, Protobuf versions and platforms.
+* After installing the PyPI package, run `pytest` in the release branch and test the ONNX basic functions like `load`, `checker.check_model` and `shape_inference.infer_shapes` on certain example ONNX model.
+
+**Test with onnxruntime**
+* onnxruntime uses ONNX as submodule so it cannot be tested with ONNX PyPI package. Instead, onnxruntime can be tested with ONNX's release commit. Update the commit ID in onnxruntime and check the status of CIs from onnxruntime.
+* To test the interaction with onnxruntime, use ONNX functions like `load`, `checker.check_model`, `shape_inference.infer_shapes`, `save` with onnxruntime functions like `InferenceSession` and `InferenceSession.run` on certain example ONNX model.
+
+**Test with ONNX converters**
+* Cooperate with the converter teams. Provide them with the produced ONNX PyPI packages and let converter teams use them with their converters to check whether there is any issue.
+
+## Upload to official PyPI
+**NOTE: Once the packages are uploaded to PyPI, you cannot overwrite it on the same PyPI instance. Please make sure everything is good on TestPyPI before uploading to PyPI**
+
+**Wheel Files**
+* Windows
+  * Same as TestPyPI, use `twine upload --verbose *.whl --repository-url https://upload.pypi.org/legacy/ -u PYPI_USERNAME -p PYPI_PASSWORD` instead.
+* Linux and Mac
+  * Similar to TestPyPI. Merge `pypi-test` branch to main branch and create a new Release with main branch and tag to trigger Travis CI with uploading PyPI packages automatically. 
+
+
+**Source Distribution**
+* Follow the same process in TestPyPI to produce the source distribution.
+* Use ``twine upload --verbose dist/* --repository-url https://upload.pypi.org/legacy/`` instead to upload to the official PyPI.
+* Test with ``pip install --index-url https://upload.pypi.org/legacy/ onnx``
+
+## After PyPI Release 
+
+**Release summary**
+* Upload the source distribution, `.tar.gz` and `.zip`, in the release summary.
+* Submit the release summary with the release tag and branch.
+
+**Announce**
+* Announce in slack, for instance, `onnx-general` channel.
+* Notify ONNX partners like converter team and runtime team.
+* Create a news by updating `js/news.json` to announce ONNX release under [onnx/onnx.github.io](https://github.com/onnx/onnx.github.io) repo.
+
+**Update conda-forge package with the new ONNX version**
+* Conda builds of ONNX are done via conda-forge, which runs infrastructure for building packages and uploading them to conda-forge. You need to submit a PR to https://github.com/conda-forge/onnx-feedstock (see https://github.com/conda-forge/onnx-feedstock/pull/1/files for an example PR.) You will need to have uploaded to PyPI already, and update the version number and tarball hash of the PyPI uploaded tarball.
+
+**Set protected branch**
```

Comment:
rel-* branches are automatically marked protected right? SO may not need this explicit step

Reply:
Yes, but `rel-*` is not a required release branch naming, right? So it might be safer to keep this section unless we set `rel-*` as a required branch naming.

Reply:
we should set rel-* as required naming (release pipelines like windows also has triggers for master and rel-*)

Reply:
Make sense. Updated in line#12:
Create a release branch (please use rel-* as the branch name)...